### DISCUSSION TITLE
Cannot start Wazuh

### DISCUSSION THREAD

**Discussion by ngotri** [Security-Onion-Solutions/securityonion#5392] [Unsupported Versions]
**Dear Team,**

I have installed the new Security Onion Server version 2.3.70 (security onion-2.3.70-WAZUH) after complete install we cant  start Wazuh, 

Please help me to advise this error

![image](https://user-images.githubusercontent.com/83899925/131951639-c2b303e9-0b47-4236-929f-aaeb10b0d89f.png)

![image](https://user-images.githubusercontent.com/83899925/131955491-7eac2666-a5ba-459f-b2df-cc7be5aa531c.png)

**Comment by lan94** [comment]
The last screenshot says to run:
`systemctl status wazuh-agent.service`

What output does it generate?

**Reply by ngotri** [reply]
Hello Lan94,

This is output after running the command

![image](https://user-images.githubusercontent.com/83899925/132015459-b8c7d228-d6e5-4f4b-8a9f-e37e8294548e.png)

**Comment by dougburks** [comment]
@ngotri What options did you choose in setup? If you chose IMPORT mode as shown in https://docs.securityonion.net/en/2.3/first-time-users.html, then Wazuh is not enabled.
